gameEngine/scenes/edit_scene.py

```diff
@@ -0,0 +1,83 @@
+import pyxel
+from .base_scene import BaseScene
+from PyxelUniversalFont import Writer
+
+class EditScene(BaseScene):
+    def __init__(self, scene_manager):
+        super().__init__(scene_manager)
+        self.font_path = "misaki_gothic.ttf"
+        self.explainfont_size = 15  # フォントサイズを指定
+        self.button_font_size = 7  # フォントサイズを指定
```

Comment:
misaki_gothicは8*8フォントなのでフォントサイズの最小値は8
```suggestion
        self.button_font_size = 8  # フォントサイズを指定
```